editor/spatial_editor_gizmos.h

```diff
@@ -248,12 +248,18 @@ class ParticlesGizmo : public EditorSpatialGizmo {
 	GDCLASS(ParticlesGizmo, EditorSpatialGizmo);
 
 	Particles *particles;
+	bool display_aabb;
 
 public:
+
 	virtual String get_handle_name(int p_idx) const;
 	virtual Variant get_handle_value(int p_idx) const;
 	virtual void set_handle(int p_idx, Camera *p_camera, const Point2 &p_point);
 	virtual void commit_handle(int p_idx, const Variant &p_restore, bool p_cancel = false);
+	virtual void _changed_callback(Object *p_changed, const char * p_property);
+
+	void set_aabb_display(bool p_display);
+	void toggle_aabb_dislpay();
```

Comment:
A little typo error

Reply:
Also this is not how we usually do, please make a getter instead and make the toggle mechanism in "_toggle_aabb_display"

Reply:
You can keep the `set_aabb_display` function, but it would be better to replace the `toggle_aabb_dislpay` by a `get_aabb_display` that returns a boolean with the current visibility state.
Then in `_toggle_aabb_display` you do `gizmo->set_aabb_display(gizmoe->get_aabb_display())`

Reply:
I see, that makes a lot of sense... I will do that :)